Add permissions section for adaptive sampling (#32234)
DataDog/documentation: content/en/tracing/trace_pipeline/adaptive_sampling.md

The configuration should take effect in 5-6 minutes, the time it takes for Datadog to observe the service's traffic pattern, compute, then apply the sampling rates. Resources that have been configured remotely display as `Configured Remote` in the **Configuration** column.

## Permissions

By default, only users with the `Datadog Admin` role can modify adaptive sampling configurations or onboard services to adaptive sampling.

> To make changes, ensure your user account is upgraded to the default `Datadog Admin` role.

Reviewer: I think the second sentence is implied by the first. 

```suggestion
By default, only users with the `Datadog Admin` role can modify adaptive sampling configurations or onboard services to adaptive sampling.
```

If your organization uses custom roles, assign your user to a custom role that includes `APM Remote Configuration Write` and `APM Service Ingest Write` [permissions.][4]

### Restrict access
Use [granular access controls][19] to manage who can modify a specific service’s adaptive sampling configuration. You can restrict access based on roles, teams or individual users.

> {{< img src="/tracing/guide/adaptive_sampling/add_restriction.png" alt="Restrict permission" style="width:100%;">}}

Reviewer: The images aren't rendering. I don't see them in the `/tracing/guide/adaptive_sampling` folder either on my local machine... Did you mean to add the images to this PR?

To restrict access:

{{< img src="/tracing/guide/adaptive_sampling/restrict_service_ingestion_permissions.png" alt="Open granular access control modal" style="width:100%;">}}

1. Open the **Permissions** section in the Ingestion Control side panel of the service.

2. Click **Restrict access**.

> 3. Select the teams, roles or users to grant access.

Reviewer: ```suggestion
3. Select the teams, roles, or users to grant access to.
```

4. Click **Add**.

> Note: Only users with the `remote_config_write` permission can restrict access to the adaptive sampling configuration of individual services.

Reviewer: ```suggestion
**Note**: Only users with the `remote_config_write` permission can restrict access to the adaptive sampling configuration of individual services.
```
